core/utils.py

```diff
@@ -17,10 +18,23 @@ def __init__(self, title, margin=0, url=None, is_title=False):
 
 def get_secretary():
     """
-    Returns the Secretary. We limit this to one user.
+    Returns the Head secretary. We limit this to one user.
     """
-    return get_user_model().objects.filter(groups__name=settings.GROUP_SECRETARY)[0]
+    obj = get_all_secretaries().first()
+    obj.email = settings.EMAIL_FROM
+    return obj
 
+def get_all_secretaries():
```

Comment:
Shouldn't this filter be filtering on GROUP_SECRETARY?

From what I can gather the 'only one person' rule now applies to the GROUP_PRIMARY_SECRETARY, so this method will be the same as get_secretary, except in a QS container. 

At the least the method name is misleading at the moment.

Reply:
Yeah this makes no sense.